tests/test-verify-volume-md.yml

```diff
@@ -32,6 +32,25 @@
           regex_escape() }}"
       when: storage_test_volume.raid_metadata_version is defined
 
+    - name: Set chunk size regex
+      set_fact:
+        storage_test_md_chunk_size_re: "{{
+          storage_test_mdadm.stdout | regex_search('Chunk Size : ([0-9]+[KMG])',
+          '\\1') }}"
+      when:
+        - storage_test_volume.raid_chunk_size is defined
+        - storage_test_volume.raid_chunk_size is not none
+        - storage_test_volume.raid_chunk_size|string != "0 B"
```

Comment:
```suggestion
        - storage_test_volume.raid_chunk_size | string != "0 B"
```
Ansible style likes to have a single space on both sides of the `|` - ansible-lint should enforce this